### PR TITLE
correcting typo in community profile for shubhank-saxena

### LINKED ISSUE #1622: [Community] Member Profile: Shubhank Saxena
**Current Behavior**

@shubhank-saxena  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @shubhank-saxena  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: shubhank-saxena
- Twitter: 19_saxena
- LinkedIn: shubhank-saxena
- Link to profile picture: https://ca.slack-edge.com/TDL4L01AR-U01H9TDUR4P-499a41baa4a2-512

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
correcting mistakes in community profile for shubhank-saxena
This PR fixes #1622 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Anita-ihuman** [comment]
@Nikhil-Ladha  Please do you know why this check keeps failing?

**Comment by leecalcote** [comment]
@Anita-ihuman that failed check: my fault, not yours. 😄

**Comment by Nikhil-Ladha** [comment]
Hopefully, it will pass.....soon 🤞

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by Nikhil-Ladha** [comment]
Ahh....just one more update is needed.... @shubhank-saxena if you could provide your location details (City/State, Country)

**Comment by shubhank-saxena** [comment]
> Ahh....just one more update is needed.... @shubhank-saxena if you could provide your location details (City/State, Country)

Sure-
It's Kota/Rajasthan, India

**Comment by Nikhil-Ladha** [comment]
> Sure-
> It's Kota/Rajasthan, India

@Anita-ihuman if you could update this as well....(Rajasthan, India)

**Review by Nikhil-Ladha — COMMENTED** [review]
The key names Github, twitter linkedin all should be in small case.

File: src/collections/members/shubhank-saxena/shubhank-saxena.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Shubhank-saxena
```
Comment (Jashpatel1) [inline src/collections/members/shubhank-saxena/shubhank-saxena.mdx:2]: Can you replace the hyphen with a blank space?

**Review by Jashpatel1 — APPROVED** [review]
Looks good

**Comment by Nikhil-Ladha** [comment]
I think you have messed up the head of the branch or created different PRs from the same branch. There are changes that are not related to this PR, please update it.

**Pull request closed without merging**